defaults/main.yml

```diff
@@ -2,5 +2,5 @@
 # Pin-Priority of NodeSource repository
 nodejs_nodesource_pin_priority: 500
 
-# 0.10 or 0.12 or 4.x
-nodejs_version: "4.2"
+# 0.10 or 0.12 and 4.x, 5.x and onward
+nodejs_version: "5.0"
```

Comment:
Hello,

You're absolutely right, will fix it.

Thank you.

On Wed, Nov 18, 2015 at 3:34 AM, Rod Vagg <notifications@github.com> wrote:

> In defaults/main.yml
> <https://github.com/nodesource/ansible-nodejs-role/pull/21#discussion_r45156477>
> :
>
> > @@ -2,5 +2,5 @@
> >  # Pin-Priority of NodeSource repository
> >  nodejs_nodesource_pin_priority: 500
> >
> > -# 0.10 or 0.12 or 4.x
> > -nodejs_version: "4.2"
> > +# 0.10 or 0.12 and 4.x, 5.x and onward
> > +nodejs_version: "5.0"
>
> I *think* it might be wise to pin this to 4.x rather than 5.x because
> those who use Ansible are more likely to be LTS type users and v5.x is only
> going to last a max of 8 months so we'll be updating this config pretty
> soon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodesource/ansible-nodejs-role/pull/21/files#r45156477>
> .
>



-- 
Manuel Tiago Pereira